src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/ccd/datamigration/controller/CcdDataMigrationController.java

```diff
@@ -52,4 +54,28 @@ public CallbackResponse migrate(
         return responseBuilder.build();
     }
 
+    @PostMapping(value = "/migrateFrc", consumes = APPLICATION_JSON_VALUE)
```

Comment:
any reason for creating a new endpoint? we typically just update the existing endpoint to avoid having to update the URL end to end

Reply:
this one was added to be able to test RPET-164 and RPET-511 in parallel to be removed in future releases